<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WStageWidget</class>
 <widget class="QWidget" name="WStageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="stageGroupBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>500</width>
     <height>640</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>640</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="patternGroupBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>171</width>
     <height>190</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>190</height>
    </size>
   </property>
   <property name="title">
    <string>配时方案表</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>配时方案：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="patternSbNo">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>16</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>周期时长：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="patternSbCircle">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>阶段配时：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="patternSbStageNo">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="patternBtnMore">
          <property name="text">
           <string>更多&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="patternFromMore">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>相 位 差：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="patternSbOffset">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>协调相位差：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="patternSbCoordinate">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QPushButton" name="patternBtnDel">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="patternBtnApp">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>应用</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="patternBtnRead">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="stageParaGroupBox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>161</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>180</height>
    </size>
   </property>
   <property name="title">
    <string>阶段配时参数</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>阶段配时：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="stageSbNo">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>阶 段 号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stageLeId">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>阶段时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="stageSbTime">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="stageFrameColor">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>5</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="stageBtnMore">
        <property name="text">
         <string>更多&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="stageFromMore">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>147</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>选项参数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stageCbOption"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnStageApp">
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>应用</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStageRead">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
